nilearn/signal.py

```diff
@@ -510,3 +515,84 @@ def clean(signals, sessions=None, detrend=True, standardize=True,
         signals *= np.sqrt(signals.shape[0])  # for unit variance
 
     return signals
+
+
+def clean_img(niimg, sessions=None, detrend=True, standardize=True,
+          confounds=None, low_pass=None, high_pass=None, t_r=2.5):
```

Comment:
Nitpick: orthodox pep8 would like you to align this line with the opening parenthesis above.